.github/workflows/ci.yml

```diff
@@ -16,6 +16,7 @@ env:
   # Pin the nightly toolchain to prevent breakage.
   # This should be occasionally updated.
   RUST_NIGHTLY_TOOLCHAIN: nightly-2023-02-20
+  MSRV: 1.63
```

Comment:
We seem to already have a msrv that we parse from the rust-toolchain below in `env`. Do we also need this?

Reply:
Good call out, will update.